### PR TITLE
Display powered by google logo

### PR DESCRIPTION
**Pull request by BradyKDavis**
Adds the desktop `Powered By Google` logo to the bottom of the list view by default. This view can be removed by adding the prop `displayPoweredByGoogle: false` (initially set to be true, as it is required in [the Google terms of service](https://developers.google.com/places/web-service/policies)). This view will not appear on consumer-defined views using the `customContainerRender` prop.

This PR addresses [this issue.](https://github.com/xuopled/react-google-places-suggest/issues/41)

Kapture 2019-03-28 at 14 14 43

### CODE REVIEW

File: src/components/List/index.test.js [hunk 1]
```diff
@@ -52,6 +69,25 @@ describe("Suggest", () => {
     expect(list.find(ListItem)).toHaveLength(1)
   })
 
+  it("should render Powered By Google logo id displayPoweredByGoogle is true", () => {
```
Comment (xuopled) [inline src/components/List/index.test.js]: little typo `id` instead of `if`

File: src/index.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ class GooglePlacesSuggest extends React.Component {
     super()
 
     this.state = {
+      displayPoweredByGoogle: props.displayPoweredByGoogle,
```
Comment (xuopled) [inline src/index.js]: Why do use the state for this?

File: package.json [hunk 3]
```diff
@@ -50,6 +50,7 @@
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "chance": "^1.0.18",
```
Comment (xuopled) [inline package.json]: Please remove this `dep`

File: src/components/List/index.test.js [hunk 4]
```diff
@@ -66,6 +102,37 @@ describe("Suggest", () => {
     expect(onSelect).toHaveBeenCalled()
   })
 
+  it("should render Powered By Google logo for custom items if displayPoweredByGoogle is true given there are no items and no text result", () => {
+    ListFixture = (
+      <List
+        displayPoweredByGoogle={true}
+        items={[]}
+        textNoResults={chance.string()}
```
Comment (xuopled) [inline src/components/List/index.test.js]: Do not use an external dev for just one random string. Use `Custom no results test` for example

File: src/index.js (hunk 2 above)
Comment (BradyKDavis) [reply src/index.js]: Moved this to props.

File: src/components/List/index.test.js (hunk 1 above)
Comment (BradyKDavis) [reply src/components/List/index.test.js]: good catch! fixed.

**Comment by BradyKDavis** [comment]
@xuopled I've run yarn install again and replaced the lock file.

**Comment by xuopled** [comment]
Thank you @BradyKDavis! 

I just published `v3.7.0` with your feature

**Comment by BradyKDavis** [comment]
@xuopled do you need to update the package.json as well to match the publish?

**Pull request merged**